### DISCUSSION TITLE
Does `expectCall` work when the `count` is zero?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [foundry-rs/foundry#7187] [Help] [answered]
i.e. like this:

```solidity
vm.expectCall(
  callee: address(someAddress),
  data: abi.encodeCall(token.transfer, (alice, 10)),
  count: 0
);
```

Would this fail the test if there's at least 1 call made?

**Comment by smol-ninja** [comment, marked as answer]
Yes, it would fail. The `count` is used as a [strict parameter](https://book.getfoundry.sh/cheatcodes/expect-call#expectcall).

**Reply by PaulRBerg** [reply]
Good to know! I've also seen this in the docs now:

> `count` can also be set to 0 to assert that a call is not made
